docs/src/components/ComponentDoc/ComponentControls/ComponentButton.tsx

```diff
@@ -8,23 +8,15 @@ interface LabelledButtonProps {
   onClick?: (event: React.SyntheticEvent) => void
 }
 
-const centered: ICSSInJSStyle = {
-  textAlign: 'center',
-}
-
 const LabelledButton = createComponent<LabelledButtonProps>({
   displayName: 'LabelledButton',
   render: ({ stardust, ...props }) => {
-    const { iconName, label, active, onClick } = props
+    const { iconName, label, onClick } = props
     return (
-      <div style={centered} onClick={onClick}>
-        <Icon
-          name={iconName}
-          styles={{ color: active ? 'green' : 'grey', marginBottom: '10px', border: 0 }}
-        />
-        <br />
-        <Text content={label} styles={{ color: active ? 'green' : 'grey' }} />
-      </div>
```

Comment:
Previously these styles were hardcoded and that give a strange result in Dark/HC themes.

### Before

![image](https://user-images.githubusercontent.com/14183168/53805140-01b4b900-3f52-11e9-959b-597981fe1b0d.png)

### After

![image](https://user-images.githubusercontent.com/14183168/53805156-0aa58a80-3f52-11e9-8e8c-bef0a18748d0.png)

Reply:
Can we fix the description of the example as well, to work well in all themes?

Reply:
It comes from `Segment`'s styles as it's not defined in other themes.

Reply:
![image](https://user-images.githubusercontent.com/14183168/53806321-4ee65a00-3f55-11e9-95c9-6e1a3576cbfe.png)

https://github.com/stardust-ui/react/pull/1014/commits/f8581bd839bce3fd940c21c417e0dda9f00825ad